### DISCUSSION TITLE
Background colors not showing up after upgrading to V2

### DISCUSSION THREAD

**Discussion by raykanani** [tailwindlabs/tailwindcss#3030] [Help] [answered]
Looks like bg-color-### is not being recognized now that I've upgrade to V2.

I have a set of cards on landing page with different colored backgrounds. None of them display the background in production. Works on my local though.
https://howwefeeling.com/

Wondering if anyone has ran into this.

**Comment by adamwathan** [comment, marked as answer]
Looks like a PurgeCSS issue, are you following the rules of [writing purgeable HTML](https://tailwindcss.com/docs/optimizing-for-production#writing-purgeable-html)?

Going to move to discussions since those utilities definitely work and this is definitely a configuration problem of some kind but still more than happy to help.

**Reply by raykanani** [reply]
Thanks Adam. Looks like that was it.
